gap/Utils.gi

```diff
@@ -354,3 +354,16 @@ function(epsilon, n, q)
     
     return rec(generatorsOfSO := generatorsOfSO, D := D, E := E);
 end);
+
+InstallGlobalFunction("MatrixGroup",
+function(gens, F)
```

Comment:
I'm a bit reluctant to change this since we have already started using the function with this syntax ...

Is this a _major_ problem?

Reply:
"Major"? No of course but. But "we are already using it" isn't really a good justification, because then one could never fix a design mistake after making it.  The major problem then would be the process: if you are unwilling to change a function after introducing it, then this does not combine well with merging PRs after only 2 hours for reviews.

And the longer you wait, the more code uses these functions, so the more work it'll be to change it. 

Finally, in this case, the change is of the easy kind: if you forgot to adjust a call site, then you'll immediately notice it once that code is run. As opposed to situations were nonsense is computed and you get an error much later on,